CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing to Bearded.Utilities
```

Comment:
Update header to refer to graphics instead

Reply:
Manual find and replace failed me :D